doc/source/dev/index.rst

```diff
@@ -0,0 +1,56 @@
+.. _scipy-development:
+
+Developer Documentation
+-----------------------
+
+If you're interested in contributing to SciPy, start here:
+
+.. toctree::
+   :maxdepth: 1
+
+   conduct/code_of_conduct
+   hacking
```

Comment:
Just to mention it here in case that's of interest: you have multiple toctree's here for the development section of the docs. In the latest version of the theme, it's not supported to give those captions and have them appear in the sidebar (see eg https://pydata-sphinx-theme.readthedocs.io/en/latest/demo/index.html), to give a bit more structure to the sidebar content.

Reply:
You mean I can add the directive `:caption:` right?

Reply:
Indeed

Reply:
Is there something else to do? It does not add the caption in the sidebar in my tests.

Reply:
Ah, sorry, I should have said "latest *devopment* version", as it's not yet released (although I plan to do that end of this week). You'll need to install from github to test it out for now (but the package is easily installable with `pip install git+https://github.com/pydata/pydata-sphinx-theme.git@master` if you want to try out)

Reply:
Ah got it, thanks! Well if you plan to make a release soon, I can update my PR with it and pin the new version as soon as it's out.

Reply:
Is there a way to just have the caption in the sidebar?

Reply:
What do you mean exactly? Only the caption without actual links?

Reply:
In the example link you posted, we can see the caption appears in the body and in the sidebar. I am wondering if we can prevent the caption from being in the body. Or maybe even being able to have a caption for the sidebar which is different from the body. This would allow to be more flexible. In the body you could have a long text linking to the caption in the sidebar.

Reply:
In the example link you posted, we can see the caption appears in the body and in the sidebar. I am wondering if we can prevent the caption from being in the body. Or maybe even being able to have a caption for the sidebar which is different from the body. This would allow to be more flexible. In the body you could have a long text linking to the caption in the sidebar.

Reply:
Ah, I see, good question. I don't think sphinx allows to have different captions depending on the context, except by making the full toctree `:hidden:` on the page itself .. (you could still add a bullet point list of links manually, to mimic the original result, but that's not very nice .. )

Reply:
ok thanks I will just slightly rewrite the thing. I don't want to introduce yet another hacky way like using CSS to mask the caption in body 😅